code/__HELPERS/verbs.dm

```diff
@@ -0,0 +1,85 @@
+/**
+  * handles adding verbs and updating the stat panel browser
+  *
+  * pass the verb type path to this instead of adding it directly to verbs so the statpanel can update
+  * Arguments:
+  * * target - Who the verb is being added to, client or mob typepath
+  * * verb - typepath to a verb, or a list of verbs, supports lists of lists
+  */
+/proc/add_verb(target, verb_or_list_to_add)
+	if(!target)
+		CRASH("add_verb called without a target")
+	if(IsAdminAdvancedProcCall())
+		return
+	var/mob/M
+	if(ismob(target))
+		M = target
+		if(!M.client)
+			return
+	else if(!istype(target, /client))
+		CRASH("add_verb called on a non-mob and non-client")
```

Comment:
```suggestion
/proc/add_verb(target, verb_or_list_to_add)
	if(IsAdminAdvancedProcCall())
		return
	if(ismob(target))
		var/mob_target = target
		if(!mob_target.client)
			return
		target = mob_target.client
	else if(!istype(target, /client))
		CRASH("add_verb called on a non-mob and non-client")
```
First check is covered by the other two.
This way you also don't need to do a check at the end.